openApiDocumentation.yml

```diff
@@ -0,0 +1,561 @@
+openapi: 3.0.0
+info:
+  title: NextGen Documentation
+  description: API for Carbon Footprint and Energy Consumption
+  version: '1.0'
+
+servers:
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/tejasujith25_gmail.com/NextGen/1.0 
+paths:
+  /v1/authorization:
+    post:
+      summary: fetch token
+      description: authorize user with emailid and password
+      tags: 
+        - Authenication Journey
+      requestBody:
+        required: true
+        content:
+          application/vnd.api+json:
+            schema:
+              $ref: "#/components/schemas/ng-authorization"
+              
+      responses:
+        200:
+          description: Authenication successfully
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  token:
+                    type: string
+                    example: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c'
+                  typeoftoken: 
+                    type: string
+                    example: bearer
+                  expiresin: 
+                    type: number
+                    example: 6000 
+                  scope: 
+                    type: string
+                    example: read admin write
+                    description: Authorization token
+        400:
```

Comment:
if you credentials are wrong then it's unauthorized. We won't return as Bad request. Remove this status code